src/utils/commandRunner.ts

```diff
@@ -31,7 +31,7 @@ export class CommandRunner {
   }> {
     let executablePath: string;
     let command: string;
-    let runEnv: NodeJS.ProcessEnv | undefined;
+    let runEnv: NodeJS.ProcessEnv = {};
```

Comment:
Per https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback and the previous comment, it should be

```suggestion
    let runEnv: NodeJS.ProcessEnv = process.env;
```

Reply:
No way. We already ruled out the use of process.env as default because it will break execution environments. Did you see the thread on #173 ?

When it comes that non-ee, I need to make some tests to figure out how ProcessEnv works as we do only want to "add" stuff, not reset them for that case.

It should be notes that what is desirable for ee, is undesirable for local, where we should assume that subprocessed should inherited current env.

Reply:
Well, then why don't we make the calling code set `{}` where it needs to be reset to "delete all environment variables for the subprocess"?